woocommerce/woocommerce
# Implement missing refund tax methods for HPOS

### All Submissions:






### Changes proposed in this Pull Request:



Implements remaining `get_total_tax_refunded` and `get_total_shipping_refunded` methods for HPOS data store.

### How to test the changes in this Pull Request:
While HPOS as active
1. Enable taxes and add a sample tax rate (also enabled for shipping).
2. Create a new order, add some products and add a shipping rate. Calculate taxes.
3. In a shell prompt, test the implementation like so:
```
$order_id = XX; //Your order ID.
$order = wc_get_order( $order_id );
$order->get_data_store()->get_total_tax_refunded( $order );
$order->get_data_store()->get_total_shipping_refunded( $order );
```

Alternatively, check unit tests and see if they make sense.

### Other information:




### FOR PR REVIEWER ONLY:
